src/devices/Common/Iot/Device/Common/GeographicPosition.cs

```diff
@@ -132,6 +132,37 @@ public static void GetDegreesMinutesSeconds(double angle, int secDigits, out dou
             }
         }
 
+        /// <summary>
+        /// Returns the given angle as degree and minutes
+        /// </summary>
+        /// <param name="angle">Input angle, in degrees</param>
+        /// <param name="minDigits">Number of digits for the minutes</param>
+        /// <param name="normalizedVal">Normalized angle value (to -180 to 180)</param>
+        /// <param name="degrees">Full degrees</param>
+        /// <param name="minutes">Minutes including requested number of digits</param>
+        public static void GetDegreesMinutes(double angle, int minDigits, out double normalizedVal, out double degrees, out double minutes)
+        {
+            angle = GeographicPositionExtensions.NormalizeAngleTo180Degrees(angle);
+            normalizedVal = angle;
+            angle = Math.Abs(angle);
```

Comment:
what happens with sign?

Reply:
It's discarded here. If you need it, you get it back in the `normalizedVal` parameter. A position is normally printed without a sign but instead with N/S or E/W preprended. So instead of `-30° 20' 23.2"` one writes `30° 20' 23.2" S` (and certainly not  `-30° 20' 23.2" S` or `30° -20' 23.2"`).